api/HTMLFormControlsCollection.json

```diff
@@ -29,10 +29,10 @@
             "version_added": true
           },
           "safari": {
-            "version_added": true
+            "version_added": "6.1"
```

Comment:
I think there's a problem of a missing interface object here. `HTMLFormControlsCollection` is the type of `form.elements`, and that was added in Safari 3 per https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement#Browser_compatibility

Reply:
I looked this up in source, `HTMLFormControlsCollection` was added in https://trac.webkit.org/changeset/135093/webkit which does map to Safari 6.1, and `form.elements` was a `HTMLCollection` before that, confirmed by http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=8800 in Safari 5.1.